src/gui/MainWindow.cpp

```diff
@@ -302,9 +302,27 @@ MainWindow::MainWindow()
     new QShortcut(Qt::CTRL + Qt::Key_PageDown, this, SLOT(selectNextDatabaseTab()));
     new QShortcut(Qt::CTRL + Qt::SHIFT + Qt::Key_Tab, this, SLOT(selectPreviousDatabaseTab()));
     new QShortcut(Qt::CTRL + Qt::Key_PageUp, this, SLOT(selectPreviousDatabaseTab()));
-    new QShortcut(Qt::ALT + Qt::Key_0, this, SLOT(selectLastDatabaseTab()));
 
-    auto shortcut = new QShortcut(Qt::ALT + Qt::Key_1, this);
+    new QShortcut(Qt::CTRL + Qt::Key_9, this, SLOT(selectLastDatabaseTab()));
```

Comment:
Now you are keeping CTRL+9 shortcut alive even with WIndows systems. Why did you remove the previous ifdef?